docs/spec_draft.md

```diff
@@ -550,6 +551,64 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.iota
+
+### Semantics
+Fills a `result` array with values in increasing order starting from zero along
+the `iota_dimension` dimension.
+
+Note: For integers, if the dimension size is larger than what the element type's
+maximum value can hold, an overflow occurs and the behavior is implementation-
+defined and one of the following:
+  * mathematical result modulo $2^n$, where n is the bit width of the result,
```

Comment:
I believe this is what the HLO interpreter is producing. We should do the experiment if not already done. Else ignore.

Reply:
Yes, the HLO interpreter has this behavior, and since this is implementation specific, I will leave the two options.